docs/en/4-0-upgrade-guide.rst

```diff
@@ -14,3 +14,24 @@ Breaking Changes
 * ``Query::isEagerLoaded()``, and ``Query::eagerLoaded()`` were removed.
   Previously these methods were inherited from ``QueryTrait`` but served no
   purpose here.
+
+
+Indexes
+================
+
+IndexRegistry has been deprecated.
+
+```php
```

Comment:
This is not markdown. RST code blocks are defined [like so](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks)